paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt

```diff
@@ -51,18 +51,10 @@ internal object ImageUtils {
 
   private const val THUMBNAIL_SIZE = 1000
 
-  /** Directory where to write the thumbnails and deltas. */
-  private val failureDir: File
-    get() {
-      val buildDirString = System.getProperty("paparazzi.build.dir")
-      val failureDir = File(buildDirString, "paparazzi/failures")
-      failureDir.mkdirs()
-      return failureDir
-    }
```

Comment:
let's refactor this in a preceding separate PR?

Reply:
nvm, moving this and previous refactoring snippets out into: https://github.com/cashapp/paparazzi/pull/1377